tools/gulp/tasks/material-release.ts

```diff
@@ -0,0 +1,53 @@
+import {task, src, dest} from 'gulp';
+import {join} from 'path';
+import {writeFileSync} from 'fs';
+import {Bundler} from 'scss-bundle';
+import {execNodeTask, sequenceTask} from '../util/task_helpers';
+import {composeRelease} from '../util/package-build';
+import {COMPONENTS_DIR, DIST_MATERIAL, DIST_RELEASES} from '../constants';
+
+// There are no type definitions available for these imports.
+const gulpRename = require('gulp-rename');
+
+// Path to the release output of material.
+const releasePath = join(DIST_RELEASES, 'material');
+// The entry-point for the scss theming bundle.
+const themingEntryPointPath = join(COMPONENTS_DIR, 'core', 'theming', '_all-theme.scss');
+// Output path for the scss theming bundle.
+const themingBundlePath = join(releasePath, '_theming.scss');
+// Matches all pre-built theme css files
+const prebuiltThemeGlob = join(DIST_MATERIAL, '**/theming/prebuilt/*.css');
+// Matches all SCSS files in the library.
+const allScssGlob = join(COMPONENTS_DIR, '**/*.scss');
+
+/**
+ * Overwrite the release task for the material package. The material release will include special
+ * files, like a bundled theming SCSS file or all prebuilt themes.
+ */
+task('material:build-release', ['material:prepare-release'], () => composeRelease('material'));
+
+/**
+ * Task that will build the material package. It will also copy all prebuilt themes and build
+ * a bundled SCSS file for theming
+ */
+task('material:prepare-release', sequenceTask(
```

Comment:
We had that before for a lot of different tasks. But I just don't feel it's necessary. Just makes it harder to remind, if something is "supposed" to be called from the CLI or not.